cobalt/loader/loader.cc

```diff
@@ -143,6 +143,11 @@ void Loader::Resume(render_tree::ResourceProvider* resource_provider) {
   if (!is_load_complete_) Start();
 }
 
+void Loader::Conceal() {
```

Comment:
Instead of Plumbing through a new `Conceal`, since `ResourceProviderStub` already signals when the ResourceProvider can't provide graphics resources (See `IsResourceProviderStub` in `image_decoder.cc`, you can instead use the existing `Resume` call to pass that information to all the decoders.

Reply:
Github made a dupe of this comment?